dts/bindings/regulator/regulator-gpio.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO-controlled voltage/current of regulators
+
+include:
+  - name: base.yaml
+  - name: regulator.yaml
+    property-allowlist:
+      - regulator-name
+      - regulator-min-microvolt
+      - regulator-max-microvolt
```

Comment:
what about boot-on, always-on, init-voltage?

Reply:
they do, this is all handled automatically by the common layer.

Reply:
sorry, I don't see where  handled automatically `init-voltage`
could you please show me ?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/regulator/regulator_common.c#LL33C2-L33C2

Reply:
thanks for a pointing on that
my bad, I've searched it in Zephyr v3.3